src/mon/ConnectionTracker.cc

```diff
@@ -82,7 +84,9 @@ bool ConnectionTracker::increase_epoch(epoch_t e)
   if (e > epoch) {
     my_reports.epoch_version = version = 0;
     my_reports.epoch = epoch = e;
-    peer_reports[rank] = my_reports;
+    if (rank >= 0) {
```

Comment:
Are we okay to bump the epoch if we aren’t filling in peer_reports with new data? I don’t remember the rules here.